region/norfair/west/Ice Beam Snake Room.json

```diff
@@ -542,6 +564,11 @@
         "h_canNavigateHeatRooms",
         {"heatFrames": 50}
       ]
+    },
+    {
+      "link": [5, 5],
```

Comment:
I don't think 5 adds anything

Reply:
So I just realized the CF at node 1 doesn't work, because of the shot blocks. So I removed that one, and in this case I think it makes the one at node 5 more useful.

Reply:
I was just pretending that you jump off the shot blocks.

The Fune fireball  can hit you as you rise up with the CF starting to heal, and that is deadly suitless.

Reply:
Jumping off the shot blocks would need some more heat frames before and after the CF (when entering through node 1). Seems like using 5 is more practical?

Added requirements to kill the Fune before doing the CF.

Reply:
There are small pros and cons to using 1 or 5.  I don't know how likely it is that you want to enter a heated room to CF and then leave through the same door.

But there is a Fune across from 5.  And the fireball reaches me during the CF.  If I don't try to delay it.

Reply:
Oh, missed the Fune there, and forgot those shots go through walls. In that case, seems more useful to scrap the node 5 strat in favor of node 1. Added extra heat frames to position at node 1.